components/centos/centos-k8s-singlenode-setup/Vagrantfile

```diff
@@ -28,11 +28,24 @@ Vagrant.configure(2) do |config|
      sudo sed -i.back '/KUBE_API_ARGS=*/c\KUBE_API_ARGS="--service_account_key_file=/etc/pki/kube-apiserver/serviceaccount.key"' /etc/kubernetes/apiserver
      sudo sed -i.back '/KUBE_CONTROLLER_MANAGER_ARGS=*/c\KUBE_CONTROLLER_MANAGER_ARGS="--service_account_private_key_file=/etc/pki/kube-apiserver/serviceaccount.key"' /etc/kubernetes/controller-manager
 
-     sudo systemctl enable etcd kube-apiserver kube-controller-manager kube-scheduler
-     sudo systemctl start etcd kube-apiserver kube-controller-manager kube-scheduler
-
-     sudo systemctl enable kube-proxy kubelet
-     sudo systemctl start kube-proxy kubelet
+     echo "setup master"
+     for service in etcd kube-apiserver kube-controller-manager kube-scheduler; do
+       echo "  enable $service"
+       sudo systemctl enable $service
+
+       echo "  start $service"
+       sudo systemctl start $service
+     done
+
+     echo "setup nodes"
+     for service in etcd kube-proxy kubelet; do
```

Comment:
Since for us master and nodes are in same VM so `etcd` service can be removed from nodes section.